crates/shared/src/zeroex_api.rs

```diff
@@ -132,6 +316,12 @@ impl DefaultZeroExApi {
     /// Default 0x API URL.
     pub const DEFAULT_URL: &'static str = "https://api.0x.org/";
 
+    /// Default 0x verifying contract.
+    /// The currently latest 0x v4 contract.
+    pub const DEFAULT_VERIFICATION_CONTRACT: H160 = H160(hex_literal::hex!(
```

Comment:
ubernit: We should have an `addr` macro:
```suggestion
    pub const DEFAULT_VERIFICATION_CONTRACT: H160 = addr!(
```